benchmark/README.md

```diff
@@ -0,0 +1,38 @@
+Microbenchmarks
+===============
+This directory contains the definition of a program that measures the timing and
+resource consumption of a test tracing scenario.
+
+The benchmark uses [Google Benchmark][1], whose source is included as a git
+submodule under [./google-benchmark][2].
```

Comment:
Good catch. No, I don't think that it will work after merge. I guess GitHub doesn't follow submodules in rendered markdown links (however, if you click the submodule in the GitHub UI, it goes to the relevant tree in the other repo).

I'll make this not a link.